landing/static/js/index.js

```diff
@@ -251,3 +256,7 @@ $(document).ready(() => {
 $('#link_platform').on('click', function () {
     window.open(process.env.DO_FRONTEND_HOST + '/system/home/' + '?lang=' + localStorage.getItem('lang')); 
 });
+
+$('#link_DO').on('click', function () {
+     window.open(process.env.DO_FRONTEND_HOST + '/system/home/' + '?lang=' + localStorage.getItem('lang')); 
```

Comment:
1 concatenation is redundant
```suggestion
     window.open(process.env.DO_FRONTEND_HOST + '/system/home/?lang=' + localStorage.getItem('lang')); 
```